src/tools/gxtest.ss

```diff
@@ -28,6 +28,13 @@
                      help: "test files or directories to execute tests in; appending /... to a directory will recursively execute or tests in it. If no arguments are passed, all tests in the current directory are executed.")))
 
 (def (gxtest-main opt)
+  ;; local package context
+  (unless (getenv "GERBIL_PATH" #f)
+    (let* ((here (path-normalize (current-directory)))
+           (gerbil-path (path-expand ".gerbil" here)))
```

Comment:
Wait, when is it ./.gerbil vs ~/.gerbil ? What's the general uniform rule?

Reply:
tooling should take local .gerbil into account, but it is not quite right for core, so now it has to happen at the tooling side where it is supported.

I do agree that we need a uniform treatment, so I opened a follow up issue: ##1133

Reply:
Made it uniform across all the tools.